localstack/services/apigateway/integration.py

```diff
@@ -802,9 +802,10 @@ def invoke(self, invocation_context: ApiInvocationContext):
                 uri,
                 result.status_code,
             )
-        # apply custom response template
+        # apply custom response template for non-proxy integration
         invocation_context.response = result
-        self.response_templates.render(invocation_context)
+        if integration["type"] != "HTTP_PROXY":
+            self.response_templates.render(invocation_context)
```

Comment:
nice catch!! love that the fix is always some little line changes but the implication and set up around it is big 😄